src/store/todo/actions.tsx

```diff
@@ -0,0 +1,38 @@
+import {
+  ITodo,
+  FetchTodoRequest,
+  FetchTodoSuccess,
+  FetchTodoSuccessPayload,
+  FetchTodoFailure,
+  FetchTodoFailurePayload,
+  AddTodo,
+} from "src/types";
+import {
+  FETCH_TODO_REQUEST,
+  FETCH_TODO_FAILURE,
+  FETCH_TODO_SUCCESS,
+  ADD_TODO,
+} from "./actionTypes";
+
+export const fetchTodoRequest = (): FetchTodoRequest => ({
+  type: FETCH_TODO_REQUEST,
+});
+
+export const fetchTodoSuccess = (
+  payload: FetchTodoSuccessPayload
```

Comment:
payload: ITodos[]

Reply:
Do you want to change the name of this type or to change on ITodo[]?